app/controllers/registration/base_controller.rb

```diff
@@ -14,7 +14,12 @@ def next_action
   def complete_registration
     track('user_registration', success: true)
     current_user.update! registration_complete: true
-    redirect_to my_modules_path, notice: t('.complete')
+    if current_user.display_whats_new
```

Comment:
The pedant in me loves leaning on the power of predicates. Could you pop a `?` at the end

Reply:
i can